proof/refine/Invariants_H.thy

```diff
@@ -1317,12 +1081,11 @@ lemma objBitsKO_Data:
   "objBitsKO (if dev then KOUserDataDevice else KOUserData) = pageBits"
   by (simp add: objBits_def objBitsKO_def word_size_def)
 
-lemmas objBits_defs = tcbBlockSizeBits_def epSizeBits_def ntfnSizeBits_def cteSizeBits_def vcpuBits_def
-lemmas untypedBits_defs = minUntypedSizeBits_def maxUntypedSizeBits_def
-lemmas objBits_simps = objBits_def objBitsKO_def word_size_def archObjSize_def
-lemmas objBits_simps' = objBits_simps objBits_defs
+(* FIXME arch-split: why are these visible outside of Arch?! *)
+thm tcbBlockSizeBits_def epSizeBits_def word_size_def
```

Comment:
I guess they always have been, because it was too much work. If we accept `word_size_def` as generic, then `epSizeBits_def` would be as well, but `tcbBlockSizeBits_def` should probably not be. I guess one thing that is happening is that this is not properly arch split in C either: additional TCB fields are just `ifdef`ed instead of actual arch fields. Doesn't mean we can't properly split it on the formalisation side, of course.

Reply:
I removed this FIXME but still think that we should examine hiding tcbBlockSizeBits. Maybe in a separate PR as that'll change the spec and probably break things.

Reply:
Happy with that.